### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by chynu**
Added info on installing dependencies/pylint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
-# MiniBot
+# MiniBot Platform
```
Comment (ScottSWu) [inline README.md:1]: pls expand

File: README.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+```
+pylint minibot
+```
+
+Install the dependencies in `requirements.txt` and `requirements-rpi.txt`.
+From the root directory, run the following line in terminal.
+
+```
+pip install -r requirements.txt
+pip install -r requirements-rpi.txt
```
Comment (ScottSWu) [inline README.md]: This is only done on the raspberry pi

File: README.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+Before working with the code, make sure that all dependencies are installed after cloning
+the repository.
+
+Install pylint to use the Python linter.
+
+```
+pip install pylint
+```
+
+To run pylint on the code, run the following line from the root directory.
```
Comment (ScottSWu) [inline README.md:28]: Put pylint in like a section about committing or making changes

**Review by ScottSWu — COMMENTED** [review]
github mobile is so easy wow

File: README.md (hunk 1 above)
Comment (chynu) [reply README.md:1]: Expanded.

File: README.md (hunk 3 above)
Comment (chynu) [reply README.md:28]: Done (?).

File: README.md (hunk 2 above)
Comment (chynu) [reply README.md]: Separated into another section.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,57 @@
-# MiniBot
+# M i n i B o t   P l a t f o r m
```
Comment (ScottSWu) [inline README.md]: 🥝

**Review by ScottSWu — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (chynu) [reply README.md]: 🍥

**Pull request merged**